src/main/kotlin/io/snyk/plugin/ui/SnykSettingsDialog.kt

```diff
@@ -322,11 +327,20 @@ class SnykSettingsDialog(
             ),
         )
 
-        val endpointDescriptionLabel =
-            JLabel(
-                "<html>Sets API endpoint to use for Snyk requests. Useful for custom Snyk setups. <br/>" +
-                    "E.g. <code>https://api.eu.snyk.io</code>.</html>",
-            ).apply { font = FontUtil.minusOne(this.font) }
+        val endpointDescriptionLabel = JLabel(
+            "<html>If you're using SSO(OAuth2), Custom Endpoint configuration is automatic. <br>" +
```

Comment:
there seems to be a space missing after `SSO`. Also, why mix-up SSO with OAuth2? I think these are different things?! Why is `Custom Endpoint` capitalized?